frontend/ui/src/app/components/side-bar/side-bar.component.html

```diff
@@ -34,254 +34,195 @@
+          class="flex items-center w-full p-2 text-base text-gray-900 transition duration-75 rounded-lg group hover:bg-gray-100 dark:text-white dark:hover:bg-gray-700"
+          aria-controls="dropdown-example">
           <fa-icon
             [icon]="faBoxesStacked"
             size="lg"
             class="pl-0.5 w-6 h-6 text-gray-400 transition duration-75 dark:text-gray-400 group-hover:text-gray-900 dark:group-hover:text-white"></fa-icon>
-          <span class="ms-3">Applications</span>
-        </a>
-      </li>
-      <li>
-        <a
-          class="flex items-center p-2 text-gray-900 rounded-lg dark:text-white hover:bg-gray-100 dark:hover:bg-gray-700 group"
-          routerLinkActive="bg-gray-100 dark:bg-gray-700"
-          routerLink="/customers">
-          <fa-icon
-            [icon]="faAddressBook"
-            size="lg"
-            class="pl-0.5 w-6 h-6 text-gray-400 transition duration-75 dark:text-gray-400 group-hover:text-gray-900 dark:group-hover:text-white"></fa-icon>
-          <span class="ms-3">Customers</span>
-        </a>
-      </li>
-      <li>
-        <a
-          (click)="sidebar.hide()"
-          routerLinkActive="bg-gray-100 dark:bg-gray-700"
-          routerLink="/deployments"
-          routerLinkActive="bg-gray-100 dark:bg-gray-700"
-          class="flex items-center p-2 text-gray-900 rounded-lg dark:text-white hover:bg-gray-100 dark:hover:bg-gray-700 group">
-          <fa-icon
-            [icon]="faServer"
-            size="lg"
-            class="pl-0.5 w-6 h-6 text-gray-400 transition duration-75 dark:text-gray-400 group-hover:text-gray-900 dark:group-hover:text-white"></fa-icon>
-          <span class="ms-3">Deployments</span>
-        </a>
-      </li>
-      <li>
-        @if (featureFlags.isRegistryEnabled$ | async) {
-          <button
-            type="button"
-            (click)="toggleArtifactsSubMenu()"
-            class="flex items-center w-full p-2 text-base text-gray-900 transition duration-75 rounded-lg group hover:bg-gray-100 dark:text-white dark:hover:bg-gray-700"
-            aria-controls="dropdown-example">
-            <fa-icon
-              [icon]="faBox"
-              size="lg"
-              class="pl-0.5 w-6 h-6 text-gray-400 transition duration-75 dark:text-gray-400 group-hover:text-gray-900 dark:group-hover:text-white"></fa-icon>
-            <span class="flex-1 ms-3 text-left rtl:text-right whitespace-nowrap">Artifacts</span>
-            <span
-              class="ml-2 bg-gray-100 text-gray-800 text-xs font-medium me-2 px-2.5 py-0.5 rounded dark:bg-gray-700 dark:text-gray-300">
-              Preview
-            </span>
-            <fa-icon [icon]="faChevronDown"></fa-icon>
-          </button>
-          @if (artifactsSubMenuOpen()) {
-            <ul class="py-2 space-y-2">
-              <li>
+          <span class="flex-1 ms-3 text-left rtl:text-right whitespace-nowrap">Agents</span>
+          <fa-icon [icon]="faChevronDown"></fa-icon>
```

Comment:
Kind of not that important but I think it should be relatively easy to rotate the chevron 180 degrees when the menu is open. something like `[class.rotate-180]="agentsSubmenuOpen()"`

and animated with `class="transition duration-300"` (or something like that) :slightly_smiling_face: